climada/hazard/wildfire.py

```
@@ -0,0 +1,1165 @@
+
+        # fire identification
+        while firms.iter_ev.any():
+            # Compute cons_id: consecutive fires in current iteration
+            self._firms_cons_days(firms)
+            # Compute clus_id: cluster identifier inside cons_id
+            self._firms_clustering(firms, res_data, self.clus_thres_firms)
+            # compute event_id
+            self._firms_fire(self.days_thres_firms, firms.cons_id.values, \
+                             firms.clus_id.values, firms.event_id.values, \
+                             firms.iter_ev.values, firms.datenum.values)
+            LOGGER.info('Remaining fires to identify: %s.', str(np.argwhere(\
+            firms.iter_ev.values).size))
+
+        # remove minor fires
+        if self.remove_minor_fires_firms:
+            firms = self._firms_remove_minor_fires(firms,
+                                                   self.minor_fire_thres_firms)
+
+        # compute brightness and fill class attributes
+        LOGGER.info('Computing intensity of %s fires.',
+                    np.unique(firms.event_id).size)
+        self._calc_brightness(firms, centroids, res_centr)
+
+
+    def hull_burned_area(self, ev_id, alpha=100.87, return_plot=False):
+        """Compute the burned area for a given fire.
+        Please note: NASA advises against calculating burned area using the
+        FIRMS data.
+
+        Algorithm used: https://pypi.org/project/alphashape/
+
+        Parameters:
+            ev_id: id of the selected fire
+            alpha (float, optional): parameter used to compute the concave hull
+            return_plot (bool, optional): indicate if the output plot of the
+                concave hull algorithm should be returned
+            info
+
+        Returns:
+            float
+        """
+        ev_idx = np.argwhere(self.event_id == ev_id).reshape(-1)[0]
+        if not ev_idx.size:
+            LOGGER.error('No event with id %s', str(ev_id))
+            raise ValueError
```

Do not use the LOGGER for error. Please raise a proper exception (and do so in all the code where LOGGER.error was used so far). Also, why convert the `ev_id` to a string instead of using the proper `%d` for integer?

```suggestion
            raise ValueError(No event with id %d found', ev_id)
```